core/src/main/java/org/infinispan/interceptors/distribution/TxDistributionInterceptor.java

```diff
@@ -285,19 +336,8 @@ private Object handleTxWriteCommand(InvocationContext ctx, WriteCommand command,
       return invokeNextInterceptor(ctx, command);
```

Comment:
Oops, I'm not sure how I managed to put this here :)

I meant this comment on line 340: 

    // FIRST pass this call up the chain. Only if it succeeds (no exceptions) locally do we attempt to distribute.